src/Testcontainers/Configurations/Modules/Databases/AzuriteTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,233 @@
+namespace DotNet.Testcontainers.Configurations
+{
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  [PublicAPI]
+  public class AzuriteTestcontainerConfiguration
+  {
+    // TODO: Add support for these options based on request
+    // azurite.cert Path to a PEM or PFX cert file.Required by HTTPS mode.
+    // azurite.key Path to a PEM key file. Required when azurite.cert points to a PEM file.
+    // azurite.pwd PFX cert password. Required when azurite.cert points to a PFX file.
+    // azurite.oauth OAuth authentication level. Candidate level values: basic.
+    // AZURITE_ACCOUNTS environment variable to set account names and keys.
+
+    /// <summary>
+    /// Default Workspace location folder path. Default is /data.
+    /// </summary>
+    [PublicAPI]
+    public const string DefaultLocation = "/data";
+
+    internal const string DefaultBlobEndpoint = "0.0.0.0";
+    internal const string DefaultQueueEndpoint = "0.0.0.0";
+    internal const string DefaultTableEndpoint = "0.0.0.0";
+
+    private const int DefaultBlobPort = 10000;
+    private const int DefaultQueuePort = 10001;
+    private const int DefaultTablePort = 10002;
+    private const string DefaultAzuriteImage = "mcr.microsoft.com/azure-storage/azurite:3.18.0";
+
+    private bool blobServiceOnlyEnabled;
+    private bool queueServiceOnlyEnabled;
+    private bool tableServiceOnlyEnabled;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzuriteTestcontainerConfiguration" /> class with default Azurite image.
+    /// </summary>
+    /// <param name="image">The Docker image.</param>
+    public AzuriteTestcontainerConfiguration()
+      : this(DefaultAzuriteImage)
+    {
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzuriteTestcontainerConfiguration" /> class.
+    /// </summary>
+    /// <param name="image">The Docker image.</param>
+    public AzuriteTestcontainerConfiguration(string image)
+    {
+      this.Image = image;
+    }
+
+    /// <summary>
+    /// Gets the Docker image.
+    /// </summary>
+    [PublicAPI]
+    public string Image { get; }
+
+    /// <summary>
+    /// Gets or sets the host Blob port.
+    /// </summary>
+    /// <remarks>
+    /// Corresponds to the container blob port of the hosted service.
+    /// </remarks>
+    [PublicAPI]
+    public int BlobPort { get; set; }
+
+    /// <summary>
+    /// Gets or sets the container Blob port.
+    /// </summary>
+    [PublicAPI]
+    public int BlobContainerPort { get; set; } = DefaultBlobPort;
+
+    /// <summary>
+    /// Gets or sets a value indicating whether Blob service should run standalone.
+    /// </summary>
+    /// <remarks>
+    /// Default value is false.
+    /// </remarks>
+    [PublicAPI]
+    public bool BlobServiceOnlyEnabled
```

Comment:
I am not sure. I created proeprties eith ONLY word because azurite runs with all services or just one based on documentation https://github.com/Azure/Azurite#run-azurite-v3-docker-image. I am not sure if there is possibility to run any service combination therefore I had strict implementation. 
I will try to look at code and documenation of Azurite if container can run any combination of services. Based on result will try to update.

Reply:
Yep, that's why I added both options. If only one or all services are a valid configuration the 2nd option convers it.

Reply:
Fixed in [31aaf9c](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/31aaf9ca336fbf76c035468a3e64378c9087ccbb)